core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -247,4 +271,175 @@ public static boolean masterHostConfigured() {
   public static String valueAsString(String value) {
     return value == null ? "(no value set)" : value;
   }
+
+  /**
+   * Returns an instance of {@link AlluxioConfiguration} with the defaults and values from
+   * alluxio-site properties.
+   *
+   * @return The default set of Alluxio properties loaded from the site-properties file
+   */
+  public static AlluxioProperties defaults() {
+
+    if (sDefaultProperties == null) {
+      synchronized (DEFAULT_PROPERTIES_LOCK) { // We don't want multiple threads to reload
```

Comment:
in production we only load properties, right? Or do we ever reload?

Reply:
In production they should only be loaded once. However during testing we reload multiple times for a few tests.

Reply:
if the method is only visible for testing purposes, you can annotate it with `@VisibleForTesting`